app/uk/gov/hmrc/catalogue/github/GithubV3ApiClient.scala

```diff
@@ -31,6 +31,8 @@ import scala.util.{Failure, Success, Try}
 trait GithubV3ApiClient {
   self : GithubEndpoints with GithubCredentialsProvider  =>
 
+  def isInternal : Boolean
```

Comment:
GitHub v3 api has no knowlege of what 'internal' means, so this probably does not belong here. We previously modelled this logic using traits and dependency injection.